travis/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -ex
+if [[ "$TRAVIS_PULL_REQUEST" != "false" ]]; then     # pull request
+    bundle exec fastlane test
+elif [[ -z "$TRAVIS_TAG" && "$TRAVIS_BRANCH" == "mPower-Android" ]]; then  # non-tag commits to master branch
+    bundle exec fastlane test
+    bundle exec fastlane assemble
+elif [[ -z "$TRAVIS_TAG" && "$TRAVIS_BRANCH" =~ ^mPower-Android-.* ]]; then # non-tag commits to stable branches
```

Comment:
Acually i see what you are referring to now.  The branch should match 'master' and '^stable-.*'.  will make that fix